blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -332,14 +333,38 @@ private static SetRollup deserializeV1SetRollup(CodedInputStream in) throws IOEx
         }
         return rollup;
     }
-    
-    private static void serializeTimer(TimerRollup rollup, byte[] buf) throws IOException {
+
+    private static void serializeV1Timer(TimerRollup rollup, byte[] buf) throws IOException {
         CodedOutputStream out = CodedOutputStream.newInstance(buf);
         timerRollupSize.update(buf.length);
         out.writeRawByte(Constants.VERSION_1_TIMER);
+
+        // sum, count, countps, avg, max, min, var
+        out.writeRawVarint64((long)rollup.getSum());
```

Comment:
@GeorgeJahad The old code is not getting changed, per se. GH diff is confusing, but the method name for `serializeTimer` has been changed to `serializeV1Timer` and the same code has been added to another method called `serializeV2Timer`, with just a change of writing `sum` as double instead of long. I wonder if we can refactor it better, instead of duplicating most of the code in another method.